src/components/BSelect/BSelect.vue

```diff
@@ -61,25 +61,43 @@ export default {
   name: 'b-select',
   validator: null,
   props: {
+    /**
+    * Component Id
+    */
     id: {
       type: String,
       required: true
     },
+    /**
+    * Array of objects to be displayed as options.
+    */
     selectOptions: {
       type: Array
     },
+    /**
+    * Use to turn on validation and to indicate field is required.
+    */
     required: {
       type: Boolean,
       required: true
     },
```

Comment:
Can this just be a boolean and not 'required'

For example ->

```js
required: {
type:Boolean
}
```

Do you want them to have to add a prop that is `required=true` or `required=false`

Reply:
I'll add this in my refactor branch